worker/src/handlers/post.ts

```diff
@@ -69,7 +72,7 @@ async function handlePost(request: Request, sentry: Toucan): Promise<Response> {
 
   delete stored.value.last_write;
 
-  if (!deepEqual(stored.value, incomingPayload)) {
+  if (!deepEqual(stored.value, JSON.parse(JSON.stringify(incomingPayload)))) {
```

Comment:
why stringify and parse?

Reply:
Because superstruct adds undefined optional keys to the object, which are not present when stringifying, added comment